ext/EffectsMixedModelsExt.jl

```diff
@@ -29,4 +29,84 @@ function Effects.effects!(reference_grid::DataFrame, model::MixedModel;
     return reference_grid
 end
 
+"""
+    effects!(reference_grid::DataFrame, model::MixedModel, boot::MixedModelBootstrap;
+             eff_col=nothing, err_col=:err, typical=mean, invlink=identity,
+             lower_col=:lower, upper_col=:upper, level=nothing)
+
+Use the results of a bootstrap to compute empirical error estimates
+(instead of using the variance-covariance matrix).
+
+!!! warning
+    This method is experimental and may change in its defaults or
+    disappear entirely in a future release!
+"""
+function Effects.effects!(reference_grid::DataFrame, model::MixedModel,
+                          boot::MixedModelBootstrap;
+                          eff_col=nothing, err_col=:err, typical=mean, invlink=identity,
+                          lower_col=:lower, upper_col=:upper,
+                          level=nothing)
+    # right now this is written for a RegressionModel and implicitly assumes
+    # the existence of an appropriate formula method
+    form = formula(model)
+    form_typical = typify(reference_grid, form, modelmatrix(model); typical=typical)
+    # we don't need to worry about pivoting / fixef rank deficiency here
+    # because the default -0.0 placeholder coefs still yield the correct predictions
+    # it's the NaNs in the vcov that create problems, but this method sidesteps
+    # the vcov computation, so we're all good!
+    X = modelcols(form_typical, reference_grid)
+    eff = X * coef(model)
+    bootdf = unstack(DataFrame(boot.β), :coefname, :β)
+    β = Vector{Float64}(undef, ncol(bootdf) - 1)
+    boot_err = mapreduce(vcat, 1:nrow(bootdf)) do iter
+        copyto!(β, @view bootdf[iter, 2:end])
+        return X * β
+    end
+    err = map(std, eachcol(boot_err))
```

Comment:
Won't `boot_err` only ever have one column? `X * β` should be a vector, unless I'm missing something.

Reply:
There was a small error here that lead to it always be a single column, but the aggregation was correct

607780dc0d4694b22accdf33a197631c8ace2351

Reply:
Checking my understanding now: Because of the transpose above, `boot_err` is an iteration × observation matrix, so `std` over each column is the standard deviation across iterations for each observation. Assuming that's correct, 👍